src/DefaultBuilder/src/WebApplicationBuilder.cs

```diff
@@ -85,13 +85,13 @@ internal WebApplicationBuilder(Assembly? callingAssembly, string[]? args = null)
         public ILoggingBuilder Logging { get; }
 
         /// <summary>
-        /// An <see cref="IHostBuilder"/> for configuring server specific properties, but not building.
+        /// An <see cref="IWebHostBuilder"/> for configuring server specific properties, but not building.
```

Comment:
I wonder if these should point to the concrete types?

Reply:
Or conversely, change the property types to be the interfaces?

Reply:
I have no horse in this race, but the comments as-is are definitely wrong with regard to the code 😄